SIPS/sip-30.md

```diff
@@ -0,0 +1,60 @@
+
+- Bloated gas estimations [Issue 243](https://github.com/Synthetixio/synthetix/issues/243)
+- Causing gas loss [Issue 243](https://github.com/Synthetixio/synthetix/issues/243)
+- Perceived errors in SNX and Synth Transfers [etherscan](https://etherscan.io/address/0xe9cf7887b93150d4f2da7dfc6d502b216438f244)
+  ![Although one or more Error Occurred Reverted Contract Execution Completed](https://user-images.githubusercontent.com/799038/69776252-943b6d80-11ef-11ea-97b5-d01f849cff8b.png)
+
+There is a lot of Dex activity happening now with SNX on [uniswap](https://uniswap.info/token/0xc011a73ee8576fb46f5e1c5751ca3b9fe0af2a6f), [Kyber](https://tracker.kyber.network/#/tokens/0xc011a73ee8576fb46f5e1c5751ca3b9fe0af2a6f), sUSD [uniswap](https://uniswap.info/token/0x57ab1ec28d129707052df4df418d58a2d46d5f51), [Kyber](https://tracker.kyber.network/#/tokens/0x57ab1ec28d129707052df4df418d58a2d46d5f51) and sETH [uniswap](https://uniswap.info/token/0x5e74c9036fb86bd7ecdcb084a0673efc32ea31cb).
+The ERC223 implementation is causing a significant cumulative gas loss trading these tokens. We aim to reduce the total gas lost / consumed trading Synthetix tokens.
+
+## Specification
+
+<!--The technical specification should describe the syntax and semantics of any new feature.-->
+
+- Removing all ERC223 is a simple code change to `ExternStateToken.sol` which is inherited by `Synthetix.sol` and `Synth.sol`
+- It will require all Synths & SNX to be redeployed but no proxy addresses will change keeping all existing token addresses.
+- The current [Depot](https://contracts.synthetix.io/Depot) will no longer be able to accept sUSD deposits effectivly putting it to its end of life.
+- A new Depot will be required which will go back to using the original ERC20 approve, transferFrom workflow. This could be an opportunity to makes some additional improvements to the Depot such as making it upgradable. Putting it behind a proxy and giving it an external state contract so its logic can be upgraded.
+
+## Rationale
+
+<!--The rationale fleshes out the specification by describing what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.-->
+
+- Removing ERC223 will no longer show the transfer errors in contract to contract transfers. [etherscan](https://etherscan.io/address/0xe9cf7887b93150d4f2da7dfc6d502b216438f244)
+- This will also save 200K gas per contract to contract transfer. [github code reference](https://github.com/Synthetixio/synthetix/blob/master/contracts/TokenFallbackCaller.sol#L52)
```

Comment:
[np] Use the actual version in the link so it works in perpetuity: https://github.com/Synthetixio/synthetix/blob/v2.14.0/contracts/TokenFallbackCaller.sol#L52

Reply:
done